dlink-client/dlink-client-1.11/src/main/java/com/dlink/executor/CustomTableEnvironmentImpl.java

```diff
@@ -311,45 +320,46 @@ public <T> void createTemporaryView(String path, DataStream<T> dataStream, Strin
 
     @Override
     public List<LineageRel> getLineage(String statement) {
-        return null;
+        LineageContext lineageContext = new LineageContext(flinkChainedProgram, this);
```

Comment:
can this method as a CustomTableEnvironment's default method implementation? it look like all flink version had the same implementation.

Reply:
It's difficult to use this method as a CustomTableEnvironment's default method implementation.